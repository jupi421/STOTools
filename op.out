-2.76125198e-14 8.89384058e-03 -2.00103326e-02 -3.46944695e-18 4.98161725e-06 9.26221250e-06 1.85848865e-02
2.76125198e+00 -1.01298885e-03 -3.58437394e-02 3.46944695e-18 1.67101677e-06 3.00794935e-06 1.87807485e-02
5.52250396e+00 -4.77247480e-03 -3.87496053e-02 -6.93889390e-18 4.94861902e-06 9.21196370e-06 1.85990119e-02
8.28375594e+00 -3.20837213e-03 -4.22090828e-02 4.85722573e-17 1.68984576e-06 3.04179461e-06 1.87856642e-02
1.10450079e+01 -5.00190954e-03 -4.00822994e-02 3.46944695e-18 4.96299400e-06 9.24008877e-06 1.85999114e-02
1.38062599e+01 -3.27567195e-03 -4.23883729e-02 -3.46944695e-18 1.69287853e-06 3.04725406e-06 1.87857759e-02
1.65675119e+01 -5.01935565e-03 -4.01022591e-02 -5.20417043e-17 4.96318214e-06 9.24044044e-06 1.85999249e-02
1.93287639e+01 -3.27883675e-03 -4.23910259e-02 -6.93889390e-18 1.69288437e-06 3.04726456e-06 1.87857779e-02
2.20900158e+01 -5.01986684e-03 -4.01026682e-02 -3.46944695e-17 4.96318177e-06 9.24043974e-06 1.85999253e-02
2.48512678e+01 -3.27891795e-03 -4.23910907e-02 -2.08166817e-17 1.69288437e-06 3.04726456e-06 1.87857779e-02
2.76125198e+01 -5.01987972e-03 -4.01026785e-02 -2.77555756e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
3.03737718e+01 -3.27892000e-03 -4.23910924e-02 1.04083409e-17 1.69288437e-06 3.04726456e-06 1.87857779e-02
3.31350238e+01 -5.01988005e-03 -4.01026788e-02 -3.81639165e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
3.58962757e+01 -3.27892005e-03 -4.23910924e-02 2.77555756e-17 1.69288437e-06 3.04726456e-06 1.87857779e-02
3.86575277e+01 -5.01988006e-03 -4.01026788e-02 -4.16333634e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
4.14187797e+01 -3.27892005e-03 -4.23910924e-02 5.20417043e-17 1.69288437e-06 3.04726456e-06 1.87857779e-02
4.41800317e+01 -5.01988006e-03 -4.01026788e-02 0.00000000e+00 4.96318176e-06 9.24043972e-06 1.85999253e-02
4.69412837e+01 -3.27892005e-03 -4.23910924e-02 6.93889390e-18 1.69288437e-06 3.04726456e-06 1.87857779e-02
4.97025356e+01 -5.01988006e-03 -4.01026788e-02 -6.24500451e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
5.24637876e+01 -3.27892005e-03 -4.23910924e-02 2.08166817e-17 1.69288437e-06 3.04726456e-06 1.87857779e-02
5.52250396e+01 -5.01988006e-03 -4.01026788e-02 -4.16333634e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
5.79862916e+01 -3.27892005e-03 -4.23910924e-02 4.85722573e-17 1.69288437e-06 3.04726456e-06 1.87857779e-02
6.07475436e+01 -5.01988006e-03 -4.01026788e-02 -6.93889390e-18 4.96318176e-06 9.24043972e-06 1.85999253e-02
6.35087956e+01 -3.27892005e-03 -4.23910924e-02 -3.12250226e-17 1.69288437e-06 3.04726456e-06 1.87857779e-02
6.62700475e+01 -5.01988006e-03 -4.01026788e-02 -4.85722573e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
6.90312995e+01 -3.27892005e-03 -4.23910924e-02 1.73472348e-17 1.69288437e-06 3.04726456e-06 1.87857779e-02
7.17925515e+01 -5.01988006e-03 -4.01026788e-02 -3.46944695e-18 4.96318176e-06 9.24043972e-06 1.85999253e-02
7.45538035e+01 -3.27892005e-03 -4.23910924e-02 1.73472348e-17 1.69288437e-06 3.04726456e-06 1.87857779e-02
7.73150555e+01 -5.01988006e-03 -4.01026788e-02 -5.55111512e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
8.00763074e+01 -3.27892005e-03 -4.23910924e-02 0.00000000e+00 1.69288437e-06 3.04726456e-06 1.87857779e-02
8.28375594e+01 -5.01988006e-03 -4.01026788e-02 -1.04083409e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
8.55988114e+01 -3.27892005e-03 -4.23910924e-02 -3.46944695e-18 1.69288437e-06 3.04726456e-06 1.87857779e-02
8.83600634e+01 -5.01988006e-03 -4.01026788e-02 -8.32667268e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
9.11213154e+01 -3.27892005e-03 -4.23910924e-02 1.38777878e-17 1.69288437e-06 3.04726456e-06 1.87857779e-02
9.38825673e+01 -5.01988006e-03 -4.01026788e-02 -5.89805982e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
9.66438193e+01 -3.27892005e-03 -4.23910924e-02 0.00000000e+00 1.69288437e-06 3.04726456e-06 1.87857779e-02
9.94050713e+01 -5.01988006e-03 -4.01026788e-02 -6.93889390e-18 4.96318176e-06 9.24043972e-06 1.85999253e-02
1.02166323e+02 -3.27892005e-03 -4.23910924e-02 0.00000000e+00 1.69288437e-06 3.04726456e-06 1.87857779e-02
1.04927575e+02 -5.01988004e-03 -4.01026788e-02 2.42861287e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
1.07688827e+02 -3.27891993e-03 -4.23910923e-02 1.73472348e-17 1.69288437e-06 3.04726456e-06 1.87857779e-02
1.10450079e+02 -5.01987927e-03 -4.01026781e-02 -6.24500451e-17 4.96318176e-06 9.24043972e-06 1.85999253e-02
1.13211331e+02 -3.27891508e-03 -4.23910878e-02 0.00000000e+00 1.69288437e-06 3.04726456e-06 1.87857779e-02
1.15972583e+02 -5.01984872e-03 -4.01026501e-02 0.00000000e+00 4.96318177e-06 9.24043974e-06 1.85999252e-02
1.18733835e+02 -3.27872252e-03 -4.23909119e-02 -2.77555756e-17 1.69288437e-06 3.04726456e-06 1.87857776e-02
1.21495087e+02 -5.01862820e-03 -4.01015488e-02 -4.51028104e-17 4.96318214e-06 9.24044044e-06 1.85999233e-02
1.24256339e+02 -3.27055213e-03 -4.23844707e-02 -2.77555756e-17 1.69287854e-06 3.04725406e-06 1.87857663e-02
1.27017591e+02 -4.95409254e-03 -4.00748547e-02 -6.93889390e-18 4.96298374e-06 9.24009908e-06 1.85998784e-02
1.29778843e+02 -2.74540944e-03 -4.23502397e-02 0.00000000e+00 1.68984680e-06 3.04179458e-06 1.87857055e-02
1.32540095e+02 -1.79739395e-03 -3.98954622e-02 -6.59194921e-17 4.94696893e-06 9.21362735e-06 1.85994993e-02
1.35301347e+02 9.77340490e-03 -3.80247966e-02 -3.12250226e-17 1.67105222e-06 3.00796592e-06 1.87813981e-02
1.38062599e+02 2.67866598e-02 -2.10969762e-02 -5.20417043e-17 4.97260310e-06 9.27125918e-06 1.85539179e-02
1.40823851e+02 1.59777841e-02 4.01673543e-02 2.08166817e-17 3.11524010e-06 1.67098414e-06 1.81746207e-02
1.43585103e+02 4.15304448e-03 4.04435719e-02 3.12250226e-17 9.37125929e-06 4.97349003e-06 1.80380516e-02
1.46346355e+02 3.84418336e-03 4.23613992e-02 4.85722573e-17 3.15029750e-06 1.68976581e-06 1.82344967e-02
1.49107607e+02 1.01220045e-03 4.06729836e-02 -1.04083409e-17 9.40127069e-06 4.99084338e-06 1.80566048e-02
1.51868859e+02 3.32744790e-03 4.24286513e-02 2.77555756e-17 3.15595211e-06 1.69279734e-06 1.82376603e-02
1.54630111e+02 9.50849795e-04 4.06904120e-02 3.46944695e-18 9.40169055e-06 4.99107625e-06 1.80569857e-02
1.57391363e+02 3.31990433e-03 4.24318126e-02 5.55111512e-17 3.15596300e-06 1.69280316e-06 1.82377074e-02
1.60152615e+02 9.49731683e-04 4.06909226e-02 -3.46944695e-18 9.40169150e-06 4.99107675e-06 1.80569926e-02
1.62913867e+02 3.31972805e-03 4.24318937e-02 -3.46944695e-18 3.15596300e-06 1.69280316e-06 1.82377085e-02
1.65675119e+02 9.49703717e-04 4.06909355e-02 3.12250226e-17 9.40169152e-06 4.99107676e-06 1.80569928e-02
1.68436371e+02 3.31972361e-03 4.24318958e-02 2.42861287e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
1.71197623e+02 9.49703013e-04 4.06909358e-02 3.12250226e-17 9.40169152e-06 4.99107676e-06 1.80569928e-02
1.73958875e+02 3.31972350e-03 4.24318958e-02 2.08166817e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
1.76720127e+02 9.49702995e-04 4.06909358e-02 -1.38777878e-17 9.40169152e-06 4.99107676e-06 1.80569928e-02
1.79481379e+02 3.31972349e-03 4.24318958e-02 -1.38777878e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
1.82242631e+02 9.49702995e-04 4.06909358e-02 3.12250226e-17 9.40169152e-06 4.99107676e-06 1.80569928e-02
1.85003883e+02 3.31972349e-03 4.24318958e-02 6.59194921e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
1.87765135e+02 9.49702995e-04 4.06909358e-02 3.46944695e-17 9.40169152e-06 4.99107676e-06 1.80569928e-02
1.90526387e+02 3.31972349e-03 4.24318958e-02 1.04083409e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
1.93287639e+02 9.49702995e-04 4.06909358e-02 -3.46944695e-18 9.40169152e-06 4.99107676e-06 1.80569928e-02
1.96048891e+02 3.31972349e-03 4.24318958e-02 2.08166817e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
1.98810143e+02 9.49702995e-04 4.06909358e-02 2.42861287e-17 9.40169152e-06 4.99107676e-06 1.80569928e-02
2.01571395e+02 3.31972349e-03 4.24318958e-02 1.73472348e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
2.04332647e+02 9.49702995e-04 4.06909358e-02 -6.93889390e-18 9.40169152e-06 4.99107676e-06 1.80569928e-02
2.07093899e+02 3.31972349e-03 4.24318958e-02 -3.46944695e-18 3.15596300e-06 1.69280316e-06 1.82377085e-02
2.09855151e+02 9.49702995e-04 4.06909358e-02 3.46944695e-17 9.40169152e-06 4.99107676e-06 1.80569928e-02
2.12616403e+02 3.31972349e-03 4.24318958e-02 2.42861287e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
2.15377654e+02 9.49702995e-04 4.06909358e-02 3.46944695e-17 9.40169152e-06 4.99107676e-06 1.80569928e-02
2.18138906e+02 3.31972349e-03 4.24318958e-02 2.42861287e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
2.20900158e+02 9.49702995e-04 4.06909358e-02 0.00000000e+00 9.40169152e-06 4.99107676e-06 1.80569928e-02
2.23661410e+02 3.31972349e-03 4.24318958e-02 -6.93889390e-18 3.15596300e-06 1.69280316e-06 1.82377085e-02
2.26422662e+02 9.49702995e-04 4.06909358e-02 3.12250226e-17 9.40169152e-06 4.99107676e-06 1.80569928e-02
2.29183914e+02 3.31972349e-03 4.24318958e-02 1.04083409e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
2.31945166e+02 9.49702995e-04 4.06909358e-02 -6.93889390e-18 9.40169152e-06 4.99107676e-06 1.80569928e-02
2.34706418e+02 3.31972349e-03 4.24318958e-02 1.73472348e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
2.37467670e+02 9.49702995e-04 4.06909358e-02 3.46944695e-17 9.40169152e-06 4.99107676e-06 1.80569928e-02
2.40228922e+02 3.31972349e-03 4.24318958e-02 2.08166817e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
2.42990174e+02 9.49703001e-04 4.06909358e-02 0.00000000e+00 9.40169152e-06 4.99107676e-06 1.80569928e-02
2.45751426e+02 3.31972354e-03 4.24318957e-02 2.42861287e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
2.48512678e+02 9.49703261e-04 4.06909350e-02 -6.93889390e-18 9.40169152e-06 4.99107676e-06 1.80569928e-02
2.51273930e+02 3.31972517e-03 4.24318909e-02 2.08166817e-17 3.15596300e-06 1.69280316e-06 1.82377085e-02
2.54035182e+02 9.49713563e-04 4.06909045e-02 -1.73472348e-17 9.40169150e-06 4.99107675e-06 1.80569927e-02
2.56796434e+02 3.31979009e-03 4.24316986e-02 -3.46944695e-18 3.15596300e-06 1.69280316e-06 1.82377077e-02
2.59557686e+02 9.50121976e-04 4.06896856e-02 2.77555756e-17 9.40169055e-06 4.99107625e-06 1.80569875e-02
2.62318938e+02 3.32228886e-03 4.24235391e-02 2.08166817e-17 3.15595211e-06 1.69279734e-06 1.82376792e-02
2.65080190e+02 9.62610298e-04 4.06252479e-02 3.12250226e-17 9.40126036e-06 4.99085369e-06 1.80569201e-02
2.67841442e+02 3.34685731e-03 4.18992148e-02 2.42861287e-17 3.15029669e-06 1.68976561e-06 1.82382150e-02
2.70602694e+02 8.92437412e-04 3.74727123e-02 3.46944695e-17 9.36959179e-06 4.97514838e-06 1.80596873e-02
2.73363946e+02 4.17395617e-03 2.93907304e-02 2.08166817e-17 3.11520475e-06 1.67096688e-06 1.82340347e-02
