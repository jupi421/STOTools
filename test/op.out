-2.76125198e-14 -nan -nan -nan -nan -nan -nan
2.76125198e+00 -nan -nan -nan -nan -nan -nan
5.52250396e+00 -nan -nan -nan -nan -nan -nan
8.28375594e+00 -nan -nan -nan -nan -nan -nan
1.10450079e+01 -nan -nan -nan -nan -nan -nan
1.38062599e+01 -nan -nan -nan -nan -nan -nan
1.65675119e+01 -nan -nan -nan -nan -nan -nan
1.93287639e+01 -nan -nan -nan -nan -nan -nan
2.20900158e+01 -nan -nan -nan -nan -nan -nan
2.48512678e+01 -nan -nan -nan -nan -nan -nan
2.76125198e+01 -nan -nan -nan -nan -nan -nan
3.03737718e+01 -nan -nan -nan -nan -nan -nan
3.31350238e+01 -nan -nan -nan -nan -nan -nan
3.58962757e+01 -nan -nan -nan -nan -nan -nan
3.86575277e+01 -nan -nan -nan -nan -nan -nan
4.14187797e+01 -nan -nan -nan -nan -nan -nan
4.41800317e+01 -nan -nan -nan -nan -nan -nan
4.69412837e+01 -nan -nan -nan -nan -nan -nan
4.97025356e+01 -nan -nan -nan -nan -nan -nan
5.24637876e+01 -nan -nan -nan -nan -nan -nan
5.52250396e+01 -nan -nan -nan -nan -nan -nan
5.79862916e+01 -nan -nan -nan -nan -nan -nan
6.07475436e+01 -nan -nan -nan -nan -nan -nan
6.35087956e+01 -nan -nan -nan -nan -nan -nan
6.62700475e+01 -nan -nan -nan -nan -nan -nan
6.90312995e+01 -nan -nan -nan -nan -nan -nan
7.17925515e+01 -nan -nan -nan -nan -nan -nan
7.45538035e+01 -nan -nan -nan -nan -nan -nan
7.73150555e+01 -nan -nan -nan -nan -nan -nan
8.00763074e+01 -nan -nan -nan -nan -nan -nan
8.28375594e+01 -nan -nan -nan -nan -nan -nan
8.55988114e+01 -nan -nan -nan -nan -nan -nan
8.83600634e+01 -nan -nan -nan -nan -nan -nan
9.11213154e+01 -nan -nan -nan -nan -nan -nan
9.38825673e+01 -nan -nan -nan -nan -nan -nan
9.66438193e+01 -nan -nan -nan -nan -nan -nan
9.94050713e+01 -nan -nan -nan -nan -nan -nan
1.02166323e+02 -nan -nan -nan -nan -nan -nan
1.04927575e+02 -nan -nan -nan -nan -nan -nan
1.07688827e+02 -nan -nan -nan -nan -nan -nan
1.10450079e+02 -nan -nan -nan -nan -nan -nan
1.13211331e+02 -nan -nan -nan -nan -nan -nan
1.15972583e+02 -nan -nan -nan -nan -nan -nan
1.18733835e+02 -nan -nan -nan -nan -nan -nan
1.21495087e+02 -nan -nan -nan -nan -nan -nan
1.24256339e+02 -nan -nan -nan -nan -nan -nan
1.27017591e+02 -nan -nan -nan -nan -nan -nan
1.29778843e+02 -nan -nan -nan -nan -nan -nan
1.32540095e+02 -nan -nan -nan -nan -nan -nan
1.35301347e+02 -nan -nan -nan -nan -nan -nan
1.38062599e+02 -nan -nan -nan -nan -nan -nan
1.40823851e+02 -nan -nan -nan -nan -nan -nan
1.43585103e+02 -nan -nan -nan -nan -nan -nan
1.46346355e+02 -nan -nan -nan -nan -nan -nan
1.49107607e+02 -nan -nan -nan -nan -nan -nan
1.51868859e+02 -nan -nan -nan -nan -nan -nan
1.54630111e+02 -nan -nan -nan -nan -nan -nan
1.57391363e+02 -nan -nan -nan -nan -nan -nan
1.60152615e+02 -nan -nan -nan -nan -nan -nan
1.62913867e+02 -nan -nan -nan -nan -nan -nan
1.65675119e+02 -nan -nan -nan -nan -nan -nan
1.68436371e+02 -nan -nan -nan -nan -nan -nan
1.71197623e+02 -nan -nan -nan -nan -nan -nan
1.73958875e+02 -nan -nan -nan -nan -nan -nan
1.76720127e+02 -nan -nan -nan -nan -nan -nan
1.79481379e+02 -nan -nan -nan -nan -nan -nan
1.82242631e+02 -nan -nan -nan -nan -nan -nan
1.85003883e+02 -nan -nan -nan -nan -nan -nan
1.87765135e+02 -nan -nan -nan -nan -nan -nan
1.90526387e+02 -nan -nan -nan -nan -nan -nan
1.93287639e+02 -nan -nan -nan -nan -nan -nan
1.96048891e+02 -nan -nan -nan -nan -nan -nan
1.98810143e+02 -nan -nan -nan -nan -nan -nan
2.01571395e+02 -nan -nan -nan -nan -nan -nan
2.04332647e+02 -nan -nan -nan -nan -nan -nan
2.07093899e+02 -nan -nan -nan -nan -nan -nan
2.09855151e+02 -nan -nan -nan -nan -nan -nan
2.12616403e+02 -nan -nan -nan -nan -nan -nan
2.15377654e+02 -nan -nan -nan -nan -nan -nan
2.18138906e+02 -nan -nan -nan -nan -nan -nan
2.20900158e+02 -nan -nan -nan -nan -nan -nan
2.23661410e+02 -nan -nan -nan -nan -nan -nan
2.26422662e+02 -nan -nan -nan -nan -nan -nan
2.29183914e+02 -nan -nan -nan -nan -nan -nan
2.31945166e+02 -nan -nan -nan -nan -nan -nan
2.34706418e+02 -nan -nan -nan -nan -nan -nan
2.37467670e+02 -nan -nan -nan -nan -nan -nan
2.40228922e+02 -nan -nan -nan -nan -nan -nan
2.42990174e+02 -nan -nan -nan -nan -nan -nan
2.45751426e+02 -nan -nan -nan -nan -nan -nan
2.48512678e+02 -nan -nan -nan -nan -nan -nan
2.51273930e+02 -nan -nan -nan -nan -nan -nan
2.54035182e+02 -nan -nan -nan -nan -nan -nan
2.56796434e+02 -nan -nan -nan -nan -nan -nan
2.59557686e+02 -nan -nan -nan -nan -nan -nan
2.62318938e+02 -nan -nan -nan -nan -nan -nan
2.65080190e+02 -nan -nan -nan -nan -nan -nan
2.67841442e+02 -nan -nan -nan -nan -nan -nan
2.70602694e+02 -nan -nan -nan -nan -nan -nan
2.73363946e+02 -nan -nan -nan -nan -nan -nan
